scripts/build_funcs.sh

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env bash
+
+
+# Helpful debug info while script runs.
+PS4='\033[1m[$0:${LINENO}] $\033[0m '
+
+BINARY=sonobuoy
+TARGET=sonobuoy
+GOTARGET=github.com/vmware-tanzu/$TARGET
+GOPATH=$(go env GOPATH)
+REGISTRY=sonobuoy
+IMAGE=$REGISTRY/$TARGET
+LINUX_ARCH=(amd64 arm64)
+WIN_ARCH=amd64
+KIND_CLUSTER=kind
+
+# Not used for pushing images, just for local building on other GOOS. Defaults to
+# grabbing from the local go env but can be set manually to avoid that requirement.
+HOST_GOOS=$(go env GOOS)
+HOST_GOARCH=$(go env GOARCH)
+
+# --tags allows detecting non-annotated tags as well as annotated ones
+GIT_VERSION=$(git describe --always --dirty --tags)
+IMAGE_VERSION=$(git describe --always --dirty --tags)
+IMAGE_TAG=$(echo $IMAGE_VERSION | cut -d. -f1,2)
+IMAGE_BRANCH=$(git rev-parse --abbrev-ref HEAD | sed 's/\///g')
+GIT_REF_SHORT=$(git rev-parse --short=8 --verify HEAD)
+GIT_REF_LONG=$(git rev-parse --verify HEAD)
+
+BUILDMNT=/go/src/$GOTARGET
+BUILD_IMAGE=golang:1.16
+AMD_IMAGE=gcr.io/distroless/static:nonroot
+ARM_IMAGE=gcr.io/distroless/static:nonroot-arm64
```

Comment:
for the record, I don't know if / how this works (I don't have `arm64` Linux machine to test if the builds are _really_ `arm64`), pretty much trusting Docker for what it says.

Reply:
Yea, for now, that's kind of the status quo; we hit that a year or two ago as well and even talked about cutting off arm support since we weren't testing it. However, a few people in the community pushed back and said they do use it and so we just count on them for signal/feedback